Neos.Ui/core/src/application/LinkTypes/PhoneNumber/PhoneNumber.tsx

```diff
@@ -61,7 +64,11 @@ export const PhoneNumber = makeLinkType<PhoneNumberLinkModel, PhoneNumberLinkOpt
     },
 
     Editor: ({model, options}: {model: Nullable<PhoneNumberLinkModel>, options: OptionalDeep<PhoneNumberLinkOptions>}) => {
+        const [codeArea, setCodeArea] = useState<string>(model?.countryCallingCode || (options?.defaultCountry ? `+${getCountryCallingCode(options?.defaultCountry).toString()}` : `+${getCountryCallingCode(getCountries()[0]).toString()}`));
+        
         const i18n = useI18n();
+        const form = useForm();
+        const prefix = `linkTypeProps.Sitegeist_Archaeopteryx:PhoneNumber`;
```

Comment:
The variable is not very useful, because you use it only once. In my opinion it also worsens the readability of your code, because I need to look for the variable to unterstand, what you are doing in line 112.